starter_code/components.js

```diff
@@ -0,0 +1,35 @@
+class Image extends React.Component{
+  constructor(){
+    super();
+    this.state = { image: "http://cdn2.business2community.com/wp-content/uploads/2016/01/QA-Danny-DeVito-008.jpg" }
```

Comment:
Remember, the only things we need to put in `state` are data that might **change** over the lifetime of this component. So, will the URL to this image change? Probably not. However, whether or not the image is hidden or shown **will** change. So maybe instead of storing the URL, we should store a boolean indicating whether or not the image is being displayed or not, like so:

```js
this.state = {
  showImage: true
}
```